runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -43,9 +44,9 @@ Now, move to the next lesson (use the `j`{normal} key to scroll down).
 
 ** To move the cursor, press the `h`, `j`, `k`, `l` keys as indicated. **
 
-         ↑
-         k          Hint: The `h`{normal} key is at the left and moves left.
-     ← h   l →            The `l`{normal} key is at the right and moves right.
+          ↑
+          k         Hint: The `h`{normal} key is at the left and moves left.
+     ← h    l →           The `l`{normal} key is at the right and moves right.
```

Comment:
why the changed alignment?

Reply:
Having the j and k slightly offset indicates the order they are on the keyboard without someone less used to it having to stop and check. As I mentioned it took me ages to not get them backwards and I think this slight change will help new users

Reply:
Ah, okay, I was mostly going by the arrows; having them not aligned tweaked my OCD ;)

Reply:
it looks a bit jank in githubs preview but it's better in a terminal. At least with fira it's better